Resources/Audio/_RMC14/Lobby/attributions.yml

```
@@ -19,8 +19,8 @@
   source: "https://discord.com/channels/1168210010233376858/1168217282988757116/1292488660150915124"
 
 - files: ["Dire_Situation.ogg"]
-  license: "CC-0"
-  copyright: "Dire Situation by GoodShowOldChap (aka ahtelyahwat, aka Rick O'Shea), made with sampled audio: Generative glitch radio loop by Rmshh -- https://freesound.org/s/744013/ -- License: Attribution 4.0"
+  license: "Custom"
+  copyright: "Dire Situation by GoodShowOldChap (aka ahtelyahwat, aka Rick O'Shea) licensed as CC0-1.0, made with sampled audio: Generative glitch radio loop by Rmshh -- https://freesound.org/s/744013/ -- License: Attribution 4.0"
```

This doesn't need to be changed, the correct format for the script is CC0-1.0
```suggestion
  license: "CC0-1.0"
  copyright: "Dire Situation by GoodShowOldChap (aka ahtelyahwat, aka Rick O'Shea), made with sampled audio: Generative glitch radio loop by Rmshh -- https://freesound.org/s/744013/ -- License: Attribution 4.0"
```

---

That works then yeah